test/traveler-test.js

```diff
@@ -4,9 +4,9 @@ const expect = chai.expect;
 import Traveler from '../src/traveler.js';
```

Comment:
Making the names for the repo classes plural is a small and subtle change, but I think it was a good one for clarity and understanding. Good job.